packages/interaction/src/interaction.js

```diff
@@ -0,0 +1,74 @@
+import Convergence from '@bigtest/convergence';
+import { $, isPropertyDescriptor } from './helpers';
+import * as interactions from './properties/interactions';
+
+/**
+ * Interaction class to perform multiple interactions with the DOM
+ * within one convergence period
+ */
+export default class Interaction {
+  /**
+   * Registers a new helper method on this interaction instance
+   * @param {String} name - the name of the method
+   * @param {Function} helper - the method function
+   */
+  static register(name, helper) {
+    if (['once', 'always', 'do', 'timeout', 'run'].includes(name)) {
```

Comment:
Should this be a constant somewhere? Curious if this will ever get out of sync & not updated if anything is added in the future.

Reply:
They will probably end up disappearing from this package and we'll just extend the `Convergence` class for `Interaction`